### PR TITLE
Write tests for `bot.utils`.

### LINKED ISSUE #604: Write unit tests for `bot/utils/__init__.py`
Write unit tests for [`bot/utils/__init__.py`](../blob/master/bot/utils/__init__.py).

## Implementation details
Please make sure to read the general information in the [meta issue](553) and the [testing README](../blob/master/tests/README.md). We are aiming for a 100% [branch coverage](https://coverage.readthedocs.io/en/stable/branch.html) for this file, but if you think that is not possible, please discuss that in this issue.

## Additional information
If you want to work on this issue, **please make sure that you get assigned to it** by one of the core devs before starting to work on it. We would like to prevent the situation that multiple people are working on the same issue. To get assigned, leave a comment showing your interesting in tackling this issue.

### PR DESCRIPTION
**Pull request by jchristgit**
Closes #604.

### CODE REVIEW

File: tests/bot/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        """Tests case insensitive key access and storage."""
+        instance = utils.CaseInsensitiveDict()
+
+        key = 'LEMON'
+        value = 'trees'
+
+        instance[key] = value
+        self.assertIn(key, instance)
+        self.assertEqual(instance.get(key), value)
+        self.assertEqual(instance.pop(key), value)
```
Comment (MarkKoz) [inline tests/bot/test_utils.py]: Should `assertNotIn` after popping.

File: tests/bot/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import unittest
+
+from bot import utils
+
+
+class CaseInsensitiveDictTests(unittest.TestCase):
+    """Tests for the `CaseInsensitiveDict` container."""
+
+    def test_case_insensitive_key_access(self):
+        """Tests case insensitive key access and storage."""
```
Comment (MarkKoz) [inline tests/bot/test_utils.py]: I don't see where in here the key is accessed using a different-cased version. All access is done through the same variable.

File: tests/bot/test_utils.py (hunk 1 above)
Comment (jchristgit) [reply tests/bot/test_utils.py]: see 9e825ed

File: tests/bot/test_utils.py (hunk 2 above)
Comment (jchristgit) [reply tests/bot/test_utils.py]: woops. see 9e825ed

**Pull request merged**